### DISCUSSION TITLE
Converting to a transfer with API

### DISCUSSION THREAD

**Discussion by soraxas** [firefly-iii/firefly-iii#5717] [Firefly III]
Hi! I've been trying to use the HTTP API endpoints to do varies things, and one of them is that I want to convert some deposit/withdrawl to a transfer as they are imported from csv and they are just internal transfer between accounts.

I have had a look at https://api-docs.firefly-iii.org/ but I could find any endpoints for the corresponding functionality in [**Automation -> Rules -> Convert the transaction to a transfer**]. Thanks for any pointer!

**Comment by JC5** [comment]
I've not yet created this endpoint. 

https://github.com/firefly-iii/firefly-iii/issues/5675
